be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/PipelineError.scala

```diff
@@ -0,0 +1,4 @@
+package ch.epfl.pop.pubsub.graph
+
+final case class PipelineError(description: String)
```

Comment:
we might need an integer code, essentially saying what kind of error this is.
This error code might fit with the protocol specs.

Reply:
Good idea! I used the names we find in the protocol so that if Céline ends up serializing this error, the process would only take one line